torch/_inductor/select_algorithm.py

```diff
@@ -1757,8 +1757,20 @@ def no_op(*args, **kwargs):
             )
 
             if timings:
-                log.debug("Timings found in cache, returning no_op")
-                return no_op
+                # compilation in precompile stage is much cheaper than that in 
+                # autotuning stage
+                if len(timings) == len(choices):
+                    log.debug("Timings found in cache, returning no_op")
+                    return no_op
+                else:
+                    # TODO: remove this branch in April 2025
+                    # added warning for debugging purpose
+                    log.warning(
```

Comment:
We should avoid adding warnings that users might see which are intended for developers. I think warning will be visible even without torch logs set. can you use info instead ?